app/components/DeleteTodo.js

```diff
@@ -10,6 +10,12 @@ class DeleteTodo extends Component {
         this.state = {}
     }
 
+    function update() {
```

Comment:
Variable "x" is incremented but not used. Consider removing this line if it serves no purpose.